qa/rgw/s3tests-branch.yaml

```diff
@@ -1,4 +1,4 @@
 overrides:
   s3tests:
-    force-branch: ceph-master
-    # git_remote: https://github.com/ceph/
+    force-branch: wip-rgw-change-err-code
+    git_remote: https://github.com/pritha-srivastava/
```

Comment:
i went ahead and merged https://github.com/ceph/s3-tests/pull/602 and https://github.com/ceph/s3-tests/pull/604 to ceph-master, so we can drop this stuff for testing

Reply:
done